.github/workflows/test_cli_cpu_openvino.yaml

```diff
@@ -47,5 +48,10 @@ jobs:
           pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
           pip install -e .[testing,openvino,diffusers,timm]
 
-      - name: Run tests
-        run: pytest tests/test_cli.py -s -k "cli and cpu and openvino"
+      # - name: Run cli tests
+      #   run: |
+      #     pytest tests/test_cli.py -s -k "cli and cpu and openvino"
+
+      - name: Run tests from example folder
```

Comment:
This takes forever because the CPU for testing `ubuntu-latest` is not the most optimal for the CI, I think. Maybe we should also update the machine that run those tests